src/gui/keymap.h

```diff
@@ -48,9 +48,10 @@ struct Key {
 // Key layout/device info class
 class KeyMap {
 public:
+    // Do not reorder these.
+    // Doing so will break the profile import/export
```

Comment:
Note for any future revamp of the profile code to not make the key map dependent on a hardcoded value.